File: Curso_logica_de_programacao/Exercicios_logica_em_C#/Menor_de_tres/Menor_de_tres.cs
using System.Globalization;

namespace Menor_de_tres;

class Program
{
    static void Main(string[] args)
    {

        CultureInfo CI = CultureInfo.InvariantCulture;

        int valor1, valor2, valor3, menor;

        Console.Write("Primeiro valor: ");
        valor1 = int.Parse(Console.ReadLine());
        Console.Write("Segundo valor: ");
        valor2 = int.Parse(Console.ReadLine());
        Console.Write("Terceiro valor: ");
        valor3 = int.Parse(Console.ReadLine());

        if (valor1 <= valor2 && valor1 <= valor3)
        {
            menor = valor1;
        }else if (valor2 <= valor3)
        {
            menor = valor2;
        }
        else
        {
            menor = valor3;
        }

        System.Console.Write("Menor = " + menor);

    }
}

File: Algoritmos_e_Logica_de_Programacao/Exercícios_logica_em_Python/Menor_de_tres/Menor_de_tres.py
x: int
y: int
z: int

x = int(input("Primeiro valor: "))
y = int(input("Segundo valor: "))
z = int(input("Terceiro valor: "))

menor: int
if x < y and x < z:
    menor = x
elif y < z:
    menor = y
else:
    menor = z


print(f"Menor = {menor}")

